src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -534,6 +581,57 @@ export default class ComparisonGroupManager extends React.Component<
         this.props.store.deleteMarkedComparisonGroups();
     }
 
+    @computed get anyThereAnySharedGroups() {
+        return (
+            this.props.store.comparisonGroups.result.filter(
+                group => group.isSharedGroup
+            ).length > 0
+        );
+    }
+
+    @computed get notificationMessage() {
+        let message: JSX.Element | undefined = undefined;
+        if (this.props.store.comparisonGroups.result.length > 0) {
+            if (this.anyThereAnySharedGroups) {
+                message = (
+                    <>
+                        <span style={{ backgroundColor: '#ffa' }}>
+                            Highlighted rows
+                        </span>{' '}
+                        are shared groups.
+                    </>
+                );
+            }
+            if (
+                !this.props.store.isLoggedIn &&
+                AppConfig.serverConfig.authenticationMethod &&
+                AppConfig.serverConfig.authenticationMethod.includes(
+                    'social_auth'
+                )
+            ) {
+                message = (
+                    <>
+                        {message}
+                        <br />
+                        The groups above will not be saved without &nbsp;
+                        <button
+                            className="btn btn-default btn-xs"
+                            onClick={() =>
+                                openSocialAuthWindow(this.props.store.appStore)
+                            }
+                        >
+                            Login
+                        </button>
+                        <br />
+                    </>
+                );
+            } else if (this.anyThereAnySharedGroups) {
+                message = <>{message} They would be saved to your profile.</>;
```

Comment:
I'm not sure I understand the meaning of this message. Can you explain it more and maybe we can figure out another way to express it?

Reply:
Updated logic and added comments